front/services/address/GooglePlaceAutocomplete.js

```diff
@@ -120,7 +120,9 @@ export default class GooglePlaceAutocomplete extends EventEmitter {
         } else if (!this._address.getCountry()) {
             this._address.setCountry('FR');
         }
-
+        if(this._state.country !== null && this._state.country.short_name !== 'FR') {
```

Comment:
I think we can use the previous IF statement:
```js
if (this._state.country && this._state.country.short_name) {                                                  
    this._address.setCountry(this._state.country.short_name);                                                 
                                                                                                              
    if (this._state.country.short_name !== 'FR') {                                                            
        this._address.setRegion(                                                                              
            this._state.administrative_area_level_1 && this._state.administrative_area_level_1.long_name || ''
        )                                                                                                     
    }                                                                                                         
} else if (!this._address.getCountry()) {                                                                     
    this._address.setCountry('FR');                                                                           
}                                                                                                            
```

Reply:
fixed